doc/developer-guide/api/functions/TSLifecycleHookAdd.en.rst

```diff
@@ -1,18 +1,14 @@
-.. Licensed to the Apache Software Foundation (ASF) under one
-   or more contributor license agreements.  See the NOTICE file
-   distributed with this work for additional information
-   regarding copyright ownership.  The ASF licenses this file
-   to you under the Apache License, Version 2.0 (the
-   "License"); you may not use this file except in compliance
-   with the License.  You may obtain a copy of the License at
+.. Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+   agreements.  See the NOTICE file distributed with this work for additional information regarding
+   copyright ownership.  The ASF licenses this file to you under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with the License.  You may obtain
+   a copy of the License at
 
```

Comment:
I'm not sure we should change the look of these license headers. I.e. better to be consistently ugly across the board. Or, if we decide to reformat it, we should reformat all of them (at least in the .en.rst). But they should all look the same IMO.

Reply:
Allright. It passes the RAT.